bftengine/src/bftengine/messages/PrePrepareMsg.cpp

```diff
@@ -75,8 +75,8 @@ PrePrepareMsg::PrePrepareMsg(ReplicaId sender,
     : MessageBase(sender,
                   MsgCode::PrePrepare,
                   spanContext.data().size(),
-                  (((size + sizeof(Header)) < maxMessageSize<PrePrepareMsg>())
-                       ? (size + sizeof(Header))
+                  (((size + sizeof(Header) + std::to_string(s).size()) < maxMessageSize<PrePrepareMsg>())
+                       ? (size + sizeof(Header) + std::to_string(s).size())
                        : maxMessageSize<PrePrepareMsg>() - spanContext.data().size()))
```

Comment:
@Liorgal28 can you please consult with @f-squirrel on the correctness of this logic here?